dockerfiles/synapse/start.sh

```diff
@@ -5,14 +5,14 @@ set -e
 sed -i "s/SERVER_NAME/${SERVER_NAME}/g" /conf/homeserver.yaml
 
 # Add the application service registration files to the homeserver.yaml config
-for filename in /appservices/*.yaml; do
+for filename in /complement/appservice/*.yaml; do
```

Comment:
This now matches: https://github.com/matrix-org/complement/blob/53e90f7502ac3c9f1cce7c057ac65562124031d5/internal/docker/deployer.go#L43